recipes/snorkel/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "snorkel" %}
+{% set version = "v0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/HazyResearch/{{ name }}/archive/{{ version }}-beta.tar.gz
+  sha256: 483eca03494255474452a97a0e13b27868f0fadaecb4bfc2c7610e28f54d87d8
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:  #https://github.com/HazyResearch/snorkel/blob/master/environment.yml
+    - python >=3.6
+    - ipywidgets =7.4.2
+    - pytorch ==1.1.0
```

Comment:
does your package `need` pytorch to import? Maybe instead of failing outright, it can output an error messge when appropriate functions are called. You can then package it on conda-forge, and gently prompt users to install pytorch using their preferred way.